### PR TITLE
test: Skip various tests when /sys is not mounted

### PR DESCRIPTION
**Pull request by daandemeyer**
When running tests in a container, /sys might not be mounted, so
let's make sure we skip tests that depend on /sys in this case.

### CODE REVIEW

File: src/oom/test-oomd-util.c [hunk 1]
```diff
@@ -8,6 +8,7 @@
 #include "fd-util.h"
 #include "fileio.h"
 #include "fs-util.h"
+#include "mountpoint-util.h"
```
Comment (bluca) [inline src/oom/test-oomd-util.c]: why is this added?

Comment (daandemeyer) [reply src/oom/test-oomd-util.c]: I don't see it anymore ;)

**Review by bluca — APPROVED** [review]
